### PR TITLE
SP-2576 TII and solution prepare patch

### PR DESCRIPTION
**Pull request by Idris0v**

### CODE REVIEW

File: testnet/CLI/Commands/index.md [hunk 1]
```diff
@@ -11,4 +11,5 @@ slug: "/cli/commands/"
 |[orders](/testnet/cli/commands/orders)|Manage Super Protocol orders|
 |[providers](/testnet/cli/commands/providers)|Manage Super Protocol providers|
 |[solutions](/testnet/cli/commands/solutions)|Prepare solutions to be deployed on Super Protocol|
+|[tii](/testnet/cli/commands/tii)|Prepare custom resource to be processed without sharing the source.|
```
Comment (0xDm1try) [inline testnet/CLI/Commands/index.md:14]: Remove the dot at the end to stick to a common style

File: testnet/CLI/Commands/orders/download-result.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+id: "cli-orders-download-result"
+title: "Download order result"
+slug: "/cli/commands/orders/download-result"
+sidebar_label: "download-result"
+---
+
+Download result of order with `id`
```
Comment (0xDm1try) [inline testnet/CLI/Commands/orders/download-result.md:8]: I'd put "Download order result with `id`"

File: testnet/CLI/Commands/index.md (hunk 1 above)
Comment (0xDm1try) [inline testnet/CLI/Commands/index.md:14]: Remove the dot at the end to stick to a common style

File: testnet/CLI/Commands/orders/index.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ spctl orders <command>
 |**Command**|**Description**|
 | :- | :- |
 |[cancel](/testnet/cli/commands/orders/cancel)|Cancel order with `id`|
+|[download-result](/testnet/cli/commands/orders/download-result)|Download result of order with `id`|
```
Comment (0xDm1try) [inline testnet/CLI/Commands/orders/index.md:20]: Same here "Download order result with `id`"

File: testnet/CLI/Commands/orders/index.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ spctl orders <command>
...
 |[get](/testnet/cli/commands/orders/get)|Display detailed information on order with `id`|
 |[list](/testnet/cli/commands/orders/list)|List orders|
-|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
\ No newline at end of file
+|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
+|[withdraw-deposit](/testnet/cli/commands/orders/download-result)|Withdraw unspent deposit from a completed order with `id`|
```
Comment (0xDm1try) [inline testnet/CLI/Commands/orders/index.md:24]: I think the path should be ending with the name of the command itself and not "download-result" in this case - "/testnet/cli/commands/orders/withdraw-deposit"
And what's the difference between commands "replenish" and "withdraw" here? To me, they've similar meaning

File: testnet/CLI/Commands/orders/index.md [hunk 5]
```diff
@@ -17,6 +17,8 @@ spctl orders <command>
 |**Command**|**Description**|
 | :- | :- |
 |[cancel](/testnet/cli/commands/orders/cancel)|Cancel order with `id`|
+|[download-result](/testnet/cli/commands/orders/download-result)|Download result of order with `id`|
 |[get](/testnet/cli/commands/orders/get)|Display detailed information on order with `id`|
 |[list](/testnet/cli/commands/orders/list)|List orders|
-|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
\ No newline at end of file
+|[replenish-deposit](/testnet/cli/commands/orders/replenish-deposit)|Withdraw unspent deposit from a completed order with `id`|
```
Comment (0xDm1try) [inline testnet/CLI/Commands/orders/index.md:23]: The description should probably go with "replenish-deposit" and not "Withdraw..." - Replenish deposit from a completed order with `id`

**Review by AgentRX — APPROVED** [review]

**Pull request merged**